### ISSUE TITLE
Reply with Different Appended Prefix not in Conversation

### ISSUE THREAD

**Issue by ShelbyCarter01** [KyleWJohnston/outlook-move-to-thread#5]
Sometimes when people respond depending on how they respond, the title that the email is filtered on will change. The current version of the macro doesn't find these variants. 

For instance: 

Rely will add "RE:"
Forward will add "FW:"

Some companies will other stuff: 

[EXTERNAL]
[Conversation]:="[EXTERNAL] RE: ..."
[Conversation]:="RE: [EXTERNAL] RE: ..."
[Conversation]:="FW: RE: [EXTERNAL] RE: ..."

[EXT]
[Conversation]:="[EXT] RE: ..."
[Conversation]:="RE: [EXT] RE: ..."
[Conversation]:="FW: RE: [EXT] RE: ..."

Then people reply and the Prefix grows. Is there anyway to drop off the Prefix to find the folder of the conversation so that the macro works?

**Comment by k4j8** [comment]
I've encountered this issue before and the recurring prefixes drive me nuts. However, my Outlook will correctly match the new email with the rest of the conversation. Do you have [conversation view](https://support.office.com/en-us/article/view-email-messages-by-conversation-0eeec76c-f59b-4834-98e6-05cfdfa9fb07) enabled? If so, is it matching the new email with the rest of the conversation/thread?

The macro works by checking the parent folder of each of the messages in the conversation, not by looking at the subject. Compared to matching by the subject line, this method is faster, simpler, and will correctly match emails that may have the same subject but be completely unrelated

**Comment by ShelbyCarter01** [comment]
Hello Kyle,

Yes, conversation view is enabled in my Inbox.

For some reason, for these emails, it doesn't show the conversation in conversation view. I have right click and select the find similar emails and select conversation. This will link the conversations together based on subject line. Not sure why these emails are treated differently.

I agree your algorithm is definitely more simple and faster. Maybe their security or certificate protocol is messing with the parent folder.

Any thoughts or clues?

Shelby

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by k4j8** [comment]
I recently got on an external email chain that now has a subject of "[EXT] Re: Re: Re: Re: Re: RE: Re: RE: Re: [original subject here]" and is broken into several conversations. I tried using `Find Related -> Messages in this Conversation` and found additional emails in the conversation but not all. However, it works by just searching the subject name.

While this project can move emails to their existing conversation, if Outlook breaks the conversation due to a subject change, then it's Outlook's bug and out of this project's scope. Sorry I can't be of more help. The only online resource I found was this unanswered Stack Overflow post: [outlook subject breaks conversation topic thread](https://stackoverflow.com/questions/40225472/outlook-subject-breaks-conversation-topic-thread).
